src/app/home-page/widget/entries/entries.component.html

```diff
@@ -1,31 +1,31 @@
 <div class="w-100">
   <h3>My Entries</h3>
-  <div *ngIf="!hasEntries; else userHasEntries">
+  <div *ngIf="!hasItems; else userHasEntries">
     <p>You have no entries</p>
   </div>
 
   <ng-template #userHasEntries>
     <mat-form-field class="w-100">
-      <input matInput [(ngModel)]="entryFilterText" (keyup)="onTextChange($event)" placeholder="Filter entries..." />
+      <input matInput [(ngModel)]="filterText" (keyup)="onTextChange($event)" placeholder="Filter entries..." />
     </mat-form-field>
 
-    <div *ngFor="let entry of myEntries; let i = index" class="w-100">
-      <a *ngIf="entry.entryType == entryTypeEnum.TOOL" [routerLink]="'/my-tools/' + entry.path" routerLinkActive="router-link-active">{{
+    <div *ngFor="let entry of myItems" class="w-100">
+      <a *ngIf="entry.entryType === entryTypeEnum.TOOL" [routerLink]="'/my-tools/' + entry.path" routerLinkActive="router-link-active">{{
         entry.path
       }}</a>
       <a
-        *ngIf="entry.entryType == entryTypeEnum.WORKFLOW"
+        *ngIf="entry.entryType === entryTypeEnum.WORKFLOW"
         [routerLink]="'/my-workflows/' + entry.path"
         routerLinkActive="router-link-active"
         >{{ entry.path }}</a
       >
       <a
-        *ngIf="entry.entryType == entryTypeEnum.SERVICE"
+        *ngIf="entry.entryType === entryTypeEnum.SERVICE"
         [routerLink]="'/my-services/' + entry.path"
         routerLinkActive="router-link-active"
         >{{ entry.path }}</a
       >
     </div>
-    <p *ngIf="myEntries && myEntries.length === 0">No matching entries</p>
+    <p *ngIf="myItems && myItems.length === 0">No matching entries</p>
```

Comment:
Shouldn't this be `!myItems || myItems.length == 0`? Or is myItems never null?

Reply:
I don't think it is ever null, just empty array

Reply:
Looking at the code, I think it starts out as undefined (line 14 in filtered-list.ts), so it probably is worth testing for.